OJP/OJP_FareSupport.xsd

```diff
@@ -137,45 +137,335 @@
 	<xs:annotation>
 		<xs:documentation>========================================== Booking ==========================================</xs:documentation>
 	</xs:annotation>
-	<xs:complexType name="BookingArrangementStructure">
+	<xs:simpleType name="ContactTypeEnumeration">
 		<xs:annotation>
-			<xs:documentation>[a more generalised form of BOOKING ARRANGEMENTS in TMv6] arrangement for booking any leg or legs of a journey </xs:documentation>
+			<xs:documentation>Allowed values for CONTACT role.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:NMTOKEN">
+			<xs:enumeration value="any"/>
+			<xs:enumeration value="information"/>
+			<xs:enumeration value="reservations"/>
+			<xs:enumeration value="lostProperty"/>
+			<xs:enumeration value="publicRelations"/>
+			<xs:enumeration value="complaints"/>
+			<xs:enumeration value="emergency"/>
+			<xs:enumeration value="other"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="PhoneType">
+		<xs:annotation>
+			<xs:documentation>Type for an International phone number. + Country + MSISDN e.g. +41 675601 etc.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:normalizedString"/>
+	</xs:simpleType>
+	<xs:simpleType name="EmailAddressType">
+		<xs:annotation>
+			<xs:documentation>Email address type a@b.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:normalizedString"/>
+	</xs:simpleType>
+	<xs:complexType name="ContactDetailsStructure" abstract="false">
+		<xs:annotation>
+			<xs:documentation>Type for contact details.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element name="BookingAgencyName" type="InternationalTextStructure" minOccurs="0">
+			<xs:element name="ContactPerson" type="InternationalTextStructure" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Name of the booking agency (contractual partner).</xs:documentation>
+					<xs:documentation>Name of contact person.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="BookingUrl" type="WebLinkStructure" minOccurs="0">
+			<xs:element name="Email" type="EmailAddressType" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>The email address of the contact.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Phone" type="PhoneType" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>URL to online booking service.</xs:documentation>
+					<xs:documentation>Contact telephone number.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="InfoUrl" type="WebLinkStructure" minOccurs="0">
+			<xs:element name="Fax" type="PhoneType" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>URL to information page.</xs:documentation>
+					<xs:documentation>Contact fax number.</xs:documentation>
```

Comment:
Do we need Fax? What happens or breaks if we let it away. 
As i now fax really used these days.

Reply:
The media exists...